### DISCUSSION TITLE
+ - * /

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#314] [Feature Requests] [answered]
even after years of patching cables , i still find "sum" and "subtract" weird and tedious to search and use. every other tool i know does it with symbols everyone knows...rather than written words. symbols are easier and faster to grasp. anyway, not that important but...

i just tried to make my own + - * / ops but cables does not let me. invalid characters. is there a way for me to fix this ?

**Comment by steam0r** [comment]
opnames need to be proper filenames, so...no :)

**Reply by wirmachenbunt** [reply]
schade marmelade

**Comment by pandrr** [comment]
yeah i want this just to be like hardcoded shortcuts in the search dialog...

**Reply by wirmachenbunt** [reply]
![image](https://user-images.githubusercontent.com/1138673/184919146-832ddf7c-e311-43a9-9d32-3a9c4209856b.png)
would be nice if this brings up at least the right op

**Comment by pandrr** [comment, marked as answer]
try it out on dev

**Reply by wirmachenbunt** [reply]
wow, super cool. that feels super fast now. great improvement
